src/dra/d_2F324.c

```diff
@@ -1,1656 +1,556 @@
-    0xFFD880AB,
-    0x0000FFE1,
-};
-
-u32 D_800CFCD8[] = {
-    0xFFD880AC,
-    0x0000FFE1,
-};
-
-u32 D_800CFCE0[] = {
-    0xFFE080AD,
-    0x0000FFE1,
-};
-
-u32 D_800CFCE8[] = {
-    0xFFE080AE,
-    0x0000FFE1,
-};
-
-u32 D_800CFCF0[] = {
-    0xFFE080AF,
-    0x0000FFE1,
-};
-
-u32 D_800CFCF8[] = {
-    0xFFE080B0,
-    0x0000FFE1,
-};
-
-u32 D_800CFD00[] = {
-    0xFFE080B1,
-    0x0000FFE1,
-};
-
-u32 D_800CFD08[] = {
-    0xFFE880B2,
-    0x0000FFE1,
-};
-
-u32 D_800CFD10[] = {
-    0xFFE080B3,
-    0x0000FFE9,
-};
-
-u32 D_800CFD18[] = {
-    0xFFE880B4,
-    0x0000FFE9,
-};
-
-u32 D_800CFD20[] = {
-    0xFFE880B5,
-    0x0000FFE1,
-};
-
-u32 D_800CFD28[] = {
-    0xFFE880B6,
-    0x0000FFE1,
-};
-
-u32 D_800CFD30[] = {
-    0xFFF080B7,
-    0x0000FFE9,
-};
-
-u32 D_800CFD38[] = {
-    0xFFF080B8,
-    0x0000FFE9,
-};
-
-u32 D_800CFD40[] = {
-    0xFFF880B9,
-    0x0000FFE9,
-};
-
-u32 D_800CFD48[] = {
-    0xFFF880BA,
-    0x0000FFE9,
-};
-
-u32 D_800CFD50[] = {
-    0xFFF080BB,
-    0x0000FFE9,
-};
-
-u32 D_800CFD58[] = {
-    0xFFF080BC,
-    0x0000FFE9,
-};
-
-u32 D_800CFD60[] = {
-    0xFFE880BD,
-    0x0000FFF9,
-};
-
-u32 D_800CFD68[] = {
-    0xFFE080BE,
-    0x0000FFF9,
-};
-
-u32 D_800CFD70[] = {
-    0xFFE080BF,
-    0x0000FFF9,
-};
-
-u32 D_800CFD78[] = {
-    0xFFE080C0,
-    0x0000FFF9,
-};
-
-u32 D_800CFD80[] = {
-    0xFFE080C1,
-    0x0000FFF1,
-};
-
-u32 D_800CFD88[] = {
-    0xFFE880C2,
-    0x0000FFF9,
-};
-
-u32 D_800CFD90[] = {
-    0xFFE880C3,
-    0x0000FFF9,
-};
-
-u32 D_800CFD98[] = {
-    0xFFE880C4,
-    0x0000FFF9,
-};
-
-u32 D_800CFDA0[] = {
-    0xFFF080C5,
-    0x0000FFF9,
-};
-
-u32 D_800CFDA8[] = {
-    0xFFF080C6,
-    0x0000FFF9,
-};
-
-u32 D_800CFDB0[] = {
-    0xFFF080C7,
-    0x0000FFF9,
-};
-
-u32 D_800CFDB8[] = {
-    0xFFF080C8,
-    0x0000FFF9,
-};
-
-u32 D_800CFDC0[] = {
-    0xFFE080C9,
-    0x0000FFE1,
-};
-
-u32 D_800CFDC8[] = {
-    0xFFD880CA,
-    0x0000FFE1,
-};
-
-u32 D_800CFDD0[] = {
-    0xFFE080CB,
-    0x0000FFE9,
-};
-
-u32 D_800CFDD8[] = {
-    0xFFE080CC,
-    0x0000FFE9,
-};
-
-u32 D_800CFDE0[] = {
-    0xFFD880CD,
-    0x0000FFD9,
-};
-
-u32 D_800CFDE8[] = {
-    0xFFE080CE,
-    0x0000FFD9,
-};
-
-u32 D_800CFDF0[] = {
-    0xFFE080CF,
-    0x0000FFD9,
-};
-
-u32 D_800CFDF8[] = {
-    0xFFE080D0,
-    0x0000FFD1,
-};
-
-u32 D_800CFE00[] = {
-    0xFFE080D1,
-    0x0000FFD1,
-};
-
-u32 D_800CFE08[] = {
-    0xFFE880D2,
-    0x0000FFF1,
-};
-
-u32 D_800CFE10[] = {
-    0xFFEE80D3,
-    0x0000FFEA,
-};
-
-u32 D_800CFE18[] = {
-    0xFFF080D4,
-    0x0000FFEA,
-};
-
-u32 D_800CFE20[] = {
-    0xFFF280D5,
-    0x0000FFEA,
-};
-
-u32 D_800CFE28[] = {
-    0xFFEB80D6,
-    0x0000FFEB,
-};
-
-u32 D_800CFE30[] = {
-    0xFFED80D7,
-    0x0000FFEB,
-};
-
-u32 D_800CFE38[] = {
-    0xFFED80D8,
-    0x0000FFEB,
-};
-
-u32 D_800CFE40[] = {
-    0xFFF480D9,
-    0x0000FFEC,
-};
+typedef struct {
+    u16 frame;
+    s16 pivotX;
+    s16 pivotY;
+    u16 clut;
+} AluFrame;
```

Comment:
Have we done any PSP functions that use AluFrame? Wondering if it's going to turn out to be an array again

Reply:
Not really, it's just for convenience I am doing this here. We have no function that uses `AluFrame`, which the struct definition is even self-contained within the C file.
I am still figuring out many things about the data, but I feel this is at least another stepping stone?